### DISCUSSION TITLE
How to solve many small files which size less than file block

### DISCUSSION THREAD

**Discussion by baofeidyz** [minio/minio#12870] [Q&A] [answered]
Hi, I want to know what the file block size is, and how to solve many small files with minio? eg, I have 100 million files, most of which in 500kb, I want to know how to reduce this part of the storage, because the file block is too large

**Comment by klauspost** [comment, marked as answer]
"file block" is defined by your file system, not by MinIO. Consult your file system documentation.

If your content is static, you can upload zip files that allow you to access individual files inside them: https://github.com/minio/minio/tree/master/docs/extensions/s3zip#perform-s3-operations-in-a-zip-content-

**Reply by baofeidyz** [reply]
thank you for your reply😊
